src/components/ChatGhostUI/ContentLoadingWrapper/index.native.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ContentLoader from 'react-content-loader/native';
```

Comment:
**question:** Could we fix this with an alias in the webpack config instead of importing the two different libraries and having two files?